### PR TITLE
Davit balabekyan java classes homework 5

### PR DESCRIPTION
**Pull request by davbalabekyan**
@vkotsinyan11

### CODE REVIEW

File: src/main/java/javaClasses/homework_5/davit_balabekyan/transport/interfaces/Printer.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package javaClasses.homework_5.davit_balabekyan.transport.interfaces;
+
+@FunctionalInterface
```
Comment (vkotsinyan11) [inline src/main/java/javaClasses/homework_5/davit_balabekyan/transport/interfaces/Printer.java:3]: Nice usage!

File: src/main/java/javaClasses/homework_5/davit_balabekyan/transport/Taxi.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import javaClasses.homework_5.davit_balabekyan.transport.interfaces.Printer;
+import javaClasses.homework_5.davit_balabekyan.transport.interfaces.Stoppable;
+import lombok.*;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode
+@ToString
```
Comment (vkotsinyan11) [inline src/main/java/javaClasses/homework_5/davit_balabekyan/transport/Taxi.java:13]: Well done! But keep mind  that overusing sometimes can be bad

**Review by vkotsinyan11 — APPROVED** [review]

File: src/main/java/javaClasses/homework_5/davit_balabekyan/transport/Taxi.java (hunk 2 above)
Comment (davbalabekyan) [reply src/main/java/javaClasses/homework_5/davit_balabekyan/transport/Taxi.java:13]: OK)

File: src/main/java/javaClasses/homework_5/davit_balabekyan/transport/interfaces/Printer.java (hunk 1 above)
Comment (davbalabekyan) [reply src/main/java/javaClasses/homework_5/davit_balabekyan/transport/interfaces/Printer.java:3]: 🆗

**Pull request merged**